packages/examples/tsconfig.jest.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "noUnusedLocals": false,
```

Comment:
// prevents error about graphql import not being used when artifacts are loaded
 
 This looks like it moved from further down, where it had this comment? Is this the reason this is being added here? Should we include the comment?

Reply:
I realized it makes more sense to have local test specific tsconfigs, instead of using a big one. I will make a small PR to improve it even further and add some explanations